VideoWeb/VideoWeb/Mappings/ParticipantToParticipantResponseMapper.cs

```diff
@@ -9,31 +9,33 @@
 
 namespace VideoWeb.Mappings
 {
-    public class ParticipantToParticipantResponseMapper : IMapTo<Participant, ParticipantResponse>
+    public class ParticipantToParticipantResponseMapper : IMapTo<Participant, Conference, ParticipantResponse>
     {
         private readonly IMapTo<LinkedParticipant, LinkedParticipantResponse> linkedParticipantMapper;
+        private readonly IMapTo<CivilianRoom, RoomSummaryResponse> roomMapper;
         public ParticipantToParticipantResponseMapper(IMapperFactory mapperFactory)
         {
             linkedParticipantMapper = mapperFactory.Get<LinkedParticipant, LinkedParticipantResponse>();
+            roomMapper = mapperFactory.Get<CivilianRoom, RoomSummaryResponse>();
         }
-        public ParticipantResponse Map(Participant input)
+        public ParticipantResponse Map(Participant participant, Conference conference)
         {
             
             var response = new ParticipantResponse()
             {
-                CaseTypeGroup = input.CaseTypeGroup,
-                CurrentRoom = null,
-                DisplayName = input.DisplayName,
-                FirstName = input.FirstName,
-                HearingRole = input.HearingRole,
-                Id = input.Id,
-                InterpreterRoom = null,
-                LastName = input.LastName,
-                Name = input.Name,
-                Representee = input.Representee,
-                Role = input.Role,
-                Status = input.ParticipantStatus,
-                LinkedParticipants = input.LinkedParticipants.Select(x => linkedParticipantMapper.Map(x)).ToList(),
+                CaseTypeGroup = participant.CaseTypeGroup,
+                CurrentRoom = null, // This cannot currently be gotten from the conference cache, the UI will keep the current room for an existing user.
```

Comment:
Have you raised a tech debt ticket for this?

Reply:
We could either add it to the cache, or change this so that the message comes from Video Api when everything is finished. The second option was discussed but has not been started. I will see if there was an actual ticket created for that.